src/nxdoc/web-ui/web-ui-functional-testing.md

```diff
@@ -61,7 +61,7 @@ You can check Web UI's plugin repository for more examples on Cucumber [feature]
 
 ## Functional Tests Framework
 
-The [functional tests framework for Web UI](https://github.com/nuxeo/nuxeo-web-ui/tree/master/packages/nuxeo-web-ui-ftest) exposes a large set of step definitions and page helpers, which is particularly useful for developing functional tests for projects contributing to Web UI. The provided steps were designed to cover most default functionality, but some of them are generic enough to be re-used on other contexts. However, you can create your own step definitions, while still being able to import and use the default page helpers provided by the framework. Should this not be enough, and you can do your own page helpers to wrap the logics of your custom elements.
+The [functional tests framework for Web UI](https://github.com/nuxeo/nuxeo-web-ui/tree/master/packages/nuxeo-web-ui-ftest) exposes a large set of step definitions and page helpers, which is particularly useful for developing functional tests for projects contributing to Web UI. The provided steps were designed to cover most default functionality, but some of them are generic enough to be re-used in other contexts. However, you can create your own step definitions, while still being able to import and use the default page helpers provided by the framework. Should this not be enough, and you can do your own page helpers to wrap the logic of your custom elements.
```

Comment:
Btw, maybe the ftest framework url should  point to our reference branch instead, wdyt? so, `https://github.com/nuxeo/nuxeo-web-ui/tree/maintenance-3.0.x/packages/nuxeo-web-ui-ftest`?